pkg/types/tuf/v0.0.1/tuf_manifest_v0_0_1_schema.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/tuf/tuf_manifest_v0_0_1_schema.json",
+    "title": "tuf targets payload v0.0.1 Schema",
+    "description": "Schema for tuf manifests",
+    "type": "object",
+    "properties": {
+        "_type": {
+            "description": "Type identifier",
+            "type": "string",
+            "enum": ["root", "targets", "snapshot", "timestamp"]
+        },
+        "version": {
+            "description": "File version",
+            "type": "integer"
+        },
+        "expires": {
+            "description": "Expiration date",
+            "type": "string",
+            "format": "date-time"
+        },
+        "signed": {
```

Comment:
Hmm, interesting! Would we want to record signatures also?

Reply:
Oh, that is hilarious. Could we internally name it something like "metadata" instead of "signed" please, so it's not confusing? 🙂

Reply:
It's gone now! The new format just stores `metadata` and `root` as the TUF JSON itself. No more indirection needed!

Reply:
https://github.com/sigstore/rekor/pull/383#discussion_r689863836